packages/block-library/src/separator/theme.scss

```diff
@@ -1,8 +1,9 @@
 .wp-block-separator {
 	border: none;
-	border-bottom: 2px solid $dark-gray-100;
+	border-bottom: 2px solid currentColor;
 	margin-left: auto;
 	margin-right: auto;
+	opacity: .4;
```

Comment:
[The WordPress Coding Standards say to use a leading zero in decimal numbers.](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values) Also, do we even need this opacity adjustment?